backend/api/v1/endpoints/connections.py

```
@@ -43,16 +54,48 @@ def list_connections(db: Session = Depends(get_db), current_user: User = Depends
+            raise HTTPException(status_code=401, detail="Unauthorized")
+
+        from core.connection_crud import get_pending_requests
+        pending_requests = get_pending_requests(db, current_user.id)
+        
+        return pending_requests
+    except Exception as e:
+        logging.error(f"Error fetching pending requests: {str(e)}")
+        raise HTTPException(status_code=500, detail="Internal Server Error")
+    
+@router.get("/user/{user_id}")
+def get_user(user_id: int, db: Session = Depends(get_db), current_user: User = Depends(get_current_user)):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_user`

(ANN201)